package.json

```diff
@@ -10,7 +10,7 @@
   "author": "Moneytree",
   "license": "MIT",
   "scripts": {
-    "build": "rm -rf dist && webpack",
+    "build": "rm -rf dist && NODE_OPTIONS=--openssl-legacy-provider webpack",
```

Comment:
see the commit message of 60fbf991faa4fedbc856eba1de9cf523932b222d for an explanation of this option.